common/workunit/workunit.cpp

```diff
@@ -3027,6 +3028,7 @@ class CWorkUnitFactory : public CInterface, implements IWorkUnitFactory, impleme
                 sortElements(iter, sortOrder.get(), NULL, NULL, unknownAttributes, elements);
                 return conn.getClear();
             }
+            virtual unsigned getTotalElements() { throw MakeStringException(-1, "not implemented"); }
```

Comment:
Do you mean that the 'total' for WUs/Queries = elements.ordinality()? The 'total' doesn't discount postfilters.